backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -80,6 +80,7 @@ private async void OnIsarRobotStatus(object? sender, MqttReceivedArgs mqttArgs)
             var provider = GetServiceProvider();
             var robotService = provider.GetRequiredService<IRobotService>();
             var isarRobotStatus = (IsarRobotStatusMessage)mqttArgs.Message;
+            Console.WriteLine($"Received RobotStatus message from ISAR: {isarRobotStatus.RobotStatus}");
```

Comment:
Remember to remove line writing to console without a logger

Reply:
Done